embark.el

```diff
@@ -257,24 +259,26 @@ Embark comes with three such indicators:
   verbose popup is shown after `embark-mixed-indicator-delay'
   seconds.
 
+- `embark-highlight-indicator', which highlights the target
+  at point.
+
 The protocol for indicator functions is as follows:
 
 When called from `embark-act', the indicator function is called
-without arguments.  The indicator function should then return a
-closure, which captures the indicator state.  The returned
-closure must accept up to three optional arguments, the action
-keymap, the targets (a list of conses of the type and value of
-each target, starting with the current one) and the prefix keys
-typed by the user so far.  The keymap, targets and prefix keys
-may be updated when cycling targets at point resulting in
-multiple calls to the closure.  When called from `embark-become',
-the indicator closure will be called with the keymap of commands
-to become, a fake target list containing a single target of type
-`embark-become' and whose value is the minibuffer input, and the
-prefix set to nil.  Note, in particular, that if an indicator
-function wishes to distinguish between `embark-act' and
-`embark-become' it should check whether the `car' of the first
-target is `embark-become'.
+without arguments. The indicator function should then return a
```

Comment:
Um, did you just reformat everything with a a single space between sentences? Please use `sentence-end-double-space = t`, like the built-in documentations strings do.

Reply:
Yes, I have disabled this insanity. I will reformat.

Reply:
I also don't like it, _except_ for docstrings, so it looks more like the built-in docs. I usually turn it on only when editing an Emacs Lisp packages (not even in my personal configuration).

Reply:
I don't even turn it on for Emacs packages. Call it a small rebellion against the Emacs authorities :-P

Reply:
> I don't even turn it on for Emacs packages. Call it a small rebellion against the Emacs authorities :-P

Just curious, do you not use the sentence motion commands?  I use them all the time and hate when they don't work because of initials or something.

Reply:
Sentence movement works for me with `sentence-end-double-space=nil`.